drdown/users/locale/pt_BR/LC_MESSAGES/django.po

```diff
@@ -376,15 +376,19 @@ msgid "Log Out"
 msgstr "Sair"
 
 #: drdown/users/templates/users/user_confirm_delete.html:8
-#: drdown/users/templates/users/user_confirm_delete.html:13
+#: drdown/users/templates/users/user_confirm_delete.html:14
 msgid "Delete"
 msgstr "Excluir"
 
 #: drdown/users/templates/users/user_confirm_delete.html:12
 msgid "Are you sure you want to delete "
 msgstr "Você tem certeza que deseja excluir "
 
-#: drdown/users/templates/users/user_confirm_delete.html:16
+#: drdown/users/templates/users/user_confirm_delete.html:13
+msgid "All data about yourself will be erase and can't be reversed."
+msgstr "Todos os seus dados serão apagados e não poderão ser revertidos."
```

Comment:
Essa frase pode dar uma evoluída e ficar assim:
"All the data about yourself will be erased and this action can't be undone."
"Todos os dados sobre você serão apagados e esta ação não pode ser desfeita."

Acredito que fica melhor